CMakeLists.txt

```diff
@@ -46,6 +46,14 @@ message(STATUS "CMAKE_INSTALL_RPATH: ${CMAKE_INSTALL_RPATH}")
 set(PLUGIN_SRC_FILES SvalinnPlugin.cpp)
 set(PLUGIN_LINK_LIBS ${CUBIT_LIBS})
 
+# HDF5 for mac
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+  SET(HDF5_USE_STATIC_LIBRARIES ON)
```

Comment:
Does use of HDF5 static libraries have any implications for how we build our DAGMC or plugin libraries?

Reply:
one need to use static HDF5 to allow o be build the plugin as a single `.dylib`

Or at least this is what I understood from my digging few months ago....